---
permalink: /
title: "About Me"
excerpt: "About Me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Hello, nice of you to stop by! I am a current epidemiologist-in-training, and am interested in how molecular epidemiology and next-generation sequencing methods can assist infectious disease research and inform public health interventions.

Research Interests...
======
The neck-breaking pace of sequencing technology development has left in its wake a vast resource of genetic and genomic data, currently totaling multiple petabases of sequence. While this information can be incredibly helpful in disease research and surveillance, the enormous scale of available sequences (and required resources for analysis) present daunting challenges on how to efficiently incorporate these data into sound epidemiologic studies. A new generation of public health researchers, trained in both epidemiology and bioinformatics, is essential to 21st century disease research to track the emergence of drug resistance, inform vaccine design, and assess the impact of interventions on pathogen populations. 

\...and Research Activities
-======


*MPH, University of Michigan School of Public Health*

My research has shown just how useful high-throughput genetic and genomic tools can be for exploring these issues. During my MPH studies at the University of Michigan, I worked in [Dr. Joseph Eisenberg's](https://sph.umich.edu/faculty-profiles/eisenberg-joseph.html) group utilizing DNA microarray platforms to detect mobile genetic elements (such as integrons and plasmids) in <i>E. coli</i> isolates from Ecuadorean birds and humans associated with small-scale production farming operations (which utilize antibiotics in poultry feed), and incorporated these data into mixed effects models to assess the impact of these practices on drug resistance in the surrounding community. Our results show not only higher levels of phenotypic resistance and genetic mobile elements in small-scale production birds compared to free-range birds, but also higher levels of genetic mobile elements and related phenotypic resistance profiles in humans associated with small-scale production birds compared to humans associated with free-range birds. These small-scale operations in developing nations, which typically house less than 100 birds per coop, appear to be more than capable of becoming a reservoir of antibiotic resistance for the humans who interact with them, and that the phenomenona of high resistance levels in livestock is not just restricted to the large-scale operations we are familiar with in the United States. [These results](https://doi.org/10.1093/aje/kwx286) have important implications for public health and policy, as small-scale production farming is increasingly viewed as [a tool for economic development](https://www.gatesnotes.com/Development/Why-I-Would-Raise-Chickens).

*PhD Candidate, Institute for Genome Sciences, University of Maryland School of Medicine*

I have since moved on to the University of Maryland Baltimore, where I am a PhD Candidate in Molecular Epidemiology working under [Dr. Joana Silva](http://www.medschool.umaryland.edu/profiles/Carneiro-da-Silva-Joana/) and [Dr. Christover V. Plowe](http://www.medschool.umaryland.edu/profiles/Plowe-Christopher/). My dissertation work at the Institute for Genome Sciences (University of Maryland School of Medicine) has focused on how molecular epidemiology and population genomics can inform the design of whole-organism malaria vaccines and aid in the interpretation of clinical trials. While whole-organism vaccines appear to offer high protection (90-100%) against malaria parasites that look like the vaccine strain, efficacy is lower against non-vaccine strains (~40-80%), suggesting that the well-established genetic diversity of this parasite hampers our ability to design effective vaccines. Multi-strain or region-specific vaccines may help improve vaccine efficacy. However, a limited amount of strains that can be adapted for vaccination purposes are available, and it is critical that that we understand how these strains differ both at the genomic and genetic level before selecting additional vaccine strains. 

<img src="images/global_pcoa.png" alt="hi" class="inline"/>
######PCoA comparing clinical P. falciparum isolates to vaccine strains (bold)######

Using both long and short read platforms, we are characterizing proposed additional vaccine strains using <i>de novo</i> assembly techniques, and are comparing them to a collection of clinical isolates from different malaria endemic regions for which we also have whole genome sequence data available using variant discovery tools. The high-quality assemblies generated through this research have been some of the first for this malaria parasite, apart from the initial reference genome, and have allowed not only for throughough genomic and genetic characterization of these strains, but also promises to reveal and resolve existing errors in the reference genome (look for preprints by late 2018/early 2019!).
