---
permalink: /
title: "About Me"
excerpt: "About Me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Hello, nice of you to stop by! I am an epidemiologist interested in how molecular epidemiology and next-generation sequencing methods can assist infectious disease research and inform public health interventions.

Research Interests...
======
The neck-breaking pace of sequencing technology development has left in its wake a vast resource of genetic and genomic data, currently totaling multiple petabases of sequence. As infectious disease researchers incorporate sequencing technologies into new fields and pathogens, we observe rapid accumulation of this data - here's the amount of sequencing data (in base pairs) accumulating for <i>Plasmodium</i> spp. since the publication of the *P. falciparum* reference genome up to last year:

<img src="images/seq-trends.png" alt="hi" class="inline"/>

While this information can be incredibly helpful in disease research and surveillance, the enormous scale of available sequences (and required resources for analysis) present daunting challenges on how to efficiently incorporate these data into sound epidemiologic studies. A new generation of public health researchers, trained in both epidemiology and bioinformatics, is essential to 21st century disease research to track the emergence of drug resistance, inform vaccine design, and assess the impact of interventions on pathogen populations. 

\...and Research Activities
======

### Genomic Epidemiology of Malaria

My research has shown just how useful high-throughput genetic and genomic tools can be for exploring these issues. My dissertation work at the Institute for Genome Sciences (University of Maryland School of Medicine) under [Dr. Joana Silva](http://www.medschool.umaryland.edu/profiles/Carneiro-da-Silva-Joana/) and [Dr. Christover V. Plowe](https://globalhealth.duke.edu/people/faculty/plowe-christopher) focused on how molecular epidemiology and population genomics can inform the design of whole-organism malaria vaccines and aid in the interpretation of clinical trials. While whole-organism vaccines appear to offer high protection (90-100%) against malaria parasites that look like the vaccine strain, efficacy is lower against non-vaccine strains (~40-80%), suggesting that the well-established genetic diversity of this parasite hampers our ability to design effective vaccines. Multi-strain or region-specific vaccines may help improve vaccine efficacy. However, a limited amount of strains that can be adapted for vaccination purposes are available, and it is critical that that we understand how these strains differ both at the genomic and genetic level before selecting additional vaccine strains. 

###### PCoA comparing clinical <i>P. falciparum</i> isolates to vaccine strains (bold)
<img src="images/global_pcoa.png" alt="hi" class="inline"/>

Using both long and short read platforms, we gererated <i>de novo</i> assemblies for strains used for vaccination and controlled human malaria challange purposes, and used these assemblies to conduct a rigorous characterization of each strain, documenting thousands of differences between strains that may impact immune recognition. In addition, we compared these strains to a collection of clinical isolates from different malaria endemic regions for which we also have whole genome sequence data available to confirm they are representative of extant parasite popultions. The high-quality assemblies generated through this research have allowed not only for throughough genomic and genetic characterization of these strains, but also reveal information on structurval variants, which have previously been difficult to directly detect with short-read technologies (manuscripts out soon!).

While whole genome sequencing technologies are great for generating as much data as possible and discovering new variants, limits on cost and low-DNA samples restrict the use of this technology for all samples. I am currently a Postdoctoral Fellow at the University of North Carolina Chapel Hill working in the Infectious Disease Epidemiology and Ecology Lab [(IDEEL)](https://www.med.unc.edu/infdis/ideel/), where we are exploring how molecular inversion probes (MIPs) can allow for targeted and efficient genotyping at lower cost compared to whole genome sequencing. MIPing samples from sub-Saharan Africa from our group have shown that this technology can also reveal population structure between and within countries, allowing a more affordable way for National Control Programs to monitor parasite population changes as interventions are rolled out.

### Don't forget about the vectors

I also have several projects related to vector genomics and vector surveys. Genotyping malaria parasites directly from the <i>Anopholes</i> mosquito host would capture a more thourough picture of circulating malaria parasites that could infect humans. In addition, many mosquitoes species are capable of transmitting a wide variety of pathogens between human and non-human hosts. Understanding the full pathogen reservoir contained in such vectors is important for understanding transmission risks for human populations. Ongoing projects include suveys of <i>Plasmodium<i/> spp. from <i>Anopholes</i> in the DRC, viral metagenomic projects for <i>Aedes aegytpi</i> (also from the DRC), and the generation of an assembly for an <i>A. aegypti</i> strain captured from Miami Beach as part of ongoing surviellence for Zika oubreaks. 

### Antibiotic resistance and small-scale poultry farming

During my MPH studies at the University of Michigan, I worked in [Dr. Joseph Eisenberg's](https://sph.umich.edu/faculty-profiles/eisenberg-joseph.html) group utilizing DNA microarray platforms to detect mobile genetic elements (such as integrons and plasmids) in <i>E. coli</i> isolates from Ecuadorean birds and humans associated with small-scale production farming operations (which utilize antibiotics in poultry feed), and incorporated these data into mixed effects models to assess the impact of these practices on drug resistance in the surrounding community. Our results show not only higher levels of phenotypic resistance and genetic mobile elements in small-scale production birds compared to free-range birds, but also higher levels of genetic mobile elements and related phenotypic resistance profiles in humans associated with small-scale production birds compared to humans associated with free-range birds. These small-scale operations in developing nations, which typically house less than 100 birds per coop, appear to be more than capable of becoming a reservoir of antibiotic resistance for the humans who interact with them, and that the phenomenona of high resistance levels in livestock is not just restricted to the large-scale operations we are familiar with in the United States. [These results](https://doi.org/10.1093/aje/kwx286) have important implications for public health and policy, as small-scale production farming is increasingly viewed as [a tool for economic development](https://www.gatesnotes.com/Development/Why-I-Would-Raise-Chickens).
